packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "a22b5dd552168a8dd925682f3b349059e10fc3d3",
-    "ref_origin" : "dcos-mesos-1.0.1"
+    "ref": "0950e8664d1b181e82dabe958d9cc1d43a5d7612",
+    "ref_origin" : "dcos-mesos-1.0.x-7b322c9"
```

Comment:
Bumped the SHA to the latest 1.0.x, so we'll stick with that branch until a 1.0.2 rc comes out.